SWLOR.Game.Server/Data/Migrations/2019-01-01.1.sql

```diff
@@ -0,0 +1,3 @@
+update dbo.LootTableItem set Resref='nw_it_gold001' where Resref='nw_it_ld001';
```

Comment:
Can you double check that this script is marked as "Embedded" in Visual Studio? Right click the file, go to properties, then switch from "Content" and select "Embedded Resource".